src/Compilers/CSharp/Test/Syntax/Parsing/FileModifierParsingTests.cs

```diff
@@ -1538,6 +1538,486 @@ class @file { }
         EOF();
     }
 
+    [Fact]
+    public void TypeNamedFile_03_CSharp10()
+    {
+        UsingNode($$"""
+            public struct file { public int item; }
+
+            public unsafe class C
+            {
+                public file _file;
```

Comment:
>public [](http://example.com/codeflow?start=16&length=6)

Consider testing the same class declaration but without the `public` modifier on the fields.

Reply:
I feel this is adequately covered by `TypeNamedRequired_CSharp10` and `TypeNamedRequired_CSharp11`